### DISCUSSION TITLE
HyperHDR v20 / Where to download/update?

### DISCUSSION THREAD

**Discussion by Cersei123** [awawa-dev/HyperSerialPico#13] [General]
Hi awawa,

thanks for your support and gift to the community.

Where do I get HyperHDR v20 from?

The latetst I can install via HomebrewChannel is v19?
What did I miss?

Thanks for your help!

**Comment by Cersei123** [comment]
Ok i see them here:
https://github.com/awawa-dev/HyperHDR/actions/runs/6922534765
But these are not for LG WebOS nor Pico, right?

**Comment by awawa-dev** [comment]
Hi
Unfortunately I don't support/maintain the HyperHDR Webos repository or build, but I think you can try OpenLGTV Discord for help.

**Comment by gnisch** [comment]
Hi 
Could You run nightly build of v 20 again, artifacts from bulid from link are expired.

Thanks in advance

**Reply by awawa-dev** [reply]
Hi
These are fine and merged with latest master:
https://github.com/awawa-dev/HyperHDR/actions/runs/7186940358
